autodispose-rxlifecycle-interop/src/main/java/com/ubercab/autodispose/rxlifecycleinterop/RXLifeCycleInterop.java

```diff
@@ -0,0 +1,54 @@
+package com.ubercab.autodispose.rxlifecycleinterop;
+
+import com.trello.rxlifecycle2.LifecycleProvider;
+import com.trello.rxlifecycle2.OutsideLifecycleException;
+import com.trello.rxlifecycle2.RxLifecycle;
+import com.uber.autodispose.AutoDispose;
+import com.uber.autodispose.LifecycleEndedException;
+import com.uber.autodispose.ScopeProvider;
+import io.reactivex.Maybe;
+
+/**
+ * Interop class for {@link RxLifecycle}. It provides static utility methods to convert {@link
+ * LifecycleProvider} to {@link ScopeProvider}
+ * <p> There are several static utility converter
```

Comment:
Nit: new line for the `<p>`, remove the space after

Reply:
Quick check - What is the code style? My formatter did it automatically.

Reply:
code style is basically the square code style - https://github.com/square/java-code-styles